test/common/router/scoped_config_impl_test.cc

```diff
@@ -52,6 +52,9 @@ TEST(HeaderValueExtractorImplDeathTest, InvalidConfig) {
   // Type not set, ASSERT only fails in debug mode.
 #if !defined(NDEBUG)
   EXPECT_DEBUG_DEATH(HeaderValueExtractorImpl{config}, "header_value_extractor is not set.");
```

Comment:
No need to use `EXPECT_DEBUG_DEATH` since there is already an `NDEBUG` guard around the statement.

Reply:
done.